### ISSUE TITLE
Typo in dmesg --help for German translation

### ISSUE THREAD

**Issue by trent2** [util-linux/util-linux#3151]
https://github.com/util-linux/util-linux/blob/a75c7a102e5dc3408ce8603a8722ba24f589971c/po/de.po#L16073

> "Einschlafen/Aufwachen will verursacht inkorrekte ctime- und iso-Zeitstempel.\n"

should be

"Einschlafen/Aufwachen wird inkorrekte ctime- und iso-Zeitstempel verursachen.\n"

**Comment by tacerus** [comment]
I think

```
"Einschlafen/Aufwachen verursacht inkorrekte ctime- und iso-Zeitstempel.\n"
```

might be even better.

**Comment by trent2** [comment]
You're right. And it seems, the text has already been corrected in [2.40-rc2](https://translationproject.org/PO-files/de/util-linux-2.40-rc2.de.po).

**Comment by tacerus** [comment]
Ah, great, then my https://github.com/util-linux/util-linux/pull/3162 is indeed not needed.
